src/Controller/Helpers/Breadcrumbs.php

```diff
@@ -0,0 +1,69 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Controller\Helpers;
+
+use App\ExternalApi\Isite\Domain\BaseIsiteObject;
+use App\ValueObject\Breadcrumb;
+
+class Breadcrumbs
+{
+    /* @var Breadcrumb[] */
+    protected $breadcrumbs = [];
+
+    /* @var BreadcrumbHelper */
+    protected $helper;
+
+    public function __construct(BreadcrumbHelper $helper, $initial = [])
+    {
+        $this->helper = $helper;
+        $this->breadcrumbs = [] + $initial;
+    }
+
+    public function forEntityAncestry($entity)
+    {
+        array_push($this->breadcrumbs, ...$this->helper->getCoreEntityAncestryBreadcrumbs($entity));
+        return $this;
+    }
+
+    public function forCategoryAncestry($category)
+    {
+        array_push($this->breadcrumbs, ...$this->helper->getCategoryAncestryBreadcrumbs($category));
+        return $this;
+    }
+
+    public function forIsiteRoute(BaseIsiteObject $isiteObj, $routeName)
+    {
+        return $this->forRoute(
+            $isiteObj->getTitle(),
+            $routeName,
+            [
+                'key' => $isiteObj->getKey(),
+                'slug' => $isiteObj->getSlug(),
+            ]
+        );
+    }
+
+    public function forNetwork($network)
+    {
+        $this->breadcrumbs[] = $this->helper->getNetworkBreadcrumb($network);
+        return $this;
+    }
+
+    public function forRoute($title, $routeName, $parameters = [])
+    {
+        $this->breadcrumbs[] = $this->helper->createBreadcrumbForRoute($title, $routeName, $parameters);
+        return $this;
+    }
+
+    public function clear()
```

Comment:
Is the `clear()` function needed?

Reply:
If it is, should probably add a type hint.
```suggestion
    public function clear(): Breadcrumbs
```